doc/whats_new/v0.21.rst

```diff
@@ -87,9 +87,12 @@ Support for Python 3.4 and below has been officially dropped.
 :mod:`sklearn.model_selection`
 ......................
 
-- |Enhancement| Method :func:`_fit_and_score` now prints train_scores when
-  `return_train_scores` is True and `verbose` > 2.
-  :issue:`12613` by :user:`Marc Torrellas <marctorrellas>`.
+- |Enhancement| Classes :class:`GridSearchCV`, :class:`RandomSearchCV`, and
+methods :func:`cross_val_score`, :func:`cross_val_predict`,
+:func:`cross_validate`, now print train scores when `return_train_scores` is
+True and `verbose` > 2. For :func:`learning_curve`, and :func:`validation_curve`
+only the latter is required.
```

Comment:
The indentation isn't right. Please keep the same indent as others entries.

Reply:
thanks for the comment, corrected